### ISSUE TITLE
Modern cmake unable to find COMCTL32

### ISSUE THREAD

**Issue by tn1997tn** [modula3/cm3#1137]
I'm building without MSYS2 as this tutorial: https://github.com/modula3/cm3/issues/1049

I noticed cmake-3.25.1-windows-x86_64 and cmake-3.24.3-windows-x86_64 both failed to find COMCTL32. The old cmake-3.16.8-win64-x64 I have on my PC has no problems with COMCTL32.

The compiler I used: https://github.com/brechtsanders/winlibs_mingw/releases/download/9.5.0-10.0.0-msvcrt-r1/winlibs-x86_64-posix-seh-gcc-9.5.0-mingw-w64msvcrt-10.0.0-r1.7z

**Comment by VictorMiasnikov** [comment]
} The old cmake-3.16.8-win64-x64 I have on my PC has no problems with COMCTL32

This is main part of description of problem.

It looks like as error in "cmake itself".

**Comment by tn1997tn** [comment]
> } The old cmake-3.16.8-win64-x64 I have on my PC has no problems with COMCTL32
> 
> This is main part of description of problem.
> 
> It looks like as error in "cmake itself".

Yes, an error in the cmake version provided by Kitware itself: https://cmake.org/download/

As I recall, there used to be older versions on Kitware's site worked, but I can't remember for sure since which version it's broke.

I also reading through the similar issue with cmake on Cygwin on this bug tracker. I confirm the cmake version provided by MSYS2 project even though being latest version still worked fine:

```
$ cmake --version
cmake version 3.25.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).

```

Cygwin must took the vanilla cmake from Kitware, the MSYS2 people should have patched their cmake.
